src/charm.py

```diff
@@ -625,14 +625,15 @@ def calicoctl(self, *args, timeout: int = 60):
         @param int timeout: If the process does not terminate after timeout seconds,
                             raise a TimeoutExpired exception
         """
-        cmd = ["/opt/calicoctl/calicoctl"] + list(args)
+        cmd = ["/opt/calicoctl/calicoctl", "--log-level=debug"] + list(args)
         env = os.environ.copy()
         env.update(self._get_calicoctl_env())
         try:
             return subprocess.check_output(cmd, env=env, stderr=subprocess.PIPE, timeout=timeout)
         except (CalledProcessError, TimeoutExpired) as e:
-            log.error(e.stderr)
-            log.error(e.output)
+            log.error("env=%s", env)
```

Comment:
Hmmm. I don't know if we want to expose the ENV here. This contains the secrets to communicate with the etcd instance.

Reply:
just the paths to the secrets -- not the secrets themselves

Reply:
it'll dump the connection string, and a few paths:

example)

```bash
ETCD_KEY_FILE=/opt/calicoctl/etcd-key
ETCD_CERT_FILE=/opt/calicoctl/etcd-cert
ETCD_CA_CERT_FILE=/opt/calicoctl/etcd-ca
ETCD_ENDPOINTS=https://10.246.154.236:2379
```

Reply:
Gotcha, just wanted to make sure the secrets are safe!